pkgs/development/python-modules/playwright/default.nix

```diff
@@ -0,0 +1,112 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, stdenv
+, fetchzip
+, greenlet
+, websockets
+, pyee
+, pillow
+, pixelmatch
+, autobahn
+, pyopenssl
+, service-identity
+, requests
+, objgraph
+}:
+
+buildPythonPackage rec {
+  pname = "playwright";
+  version = "1.21.0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "playwright-python";
+    rev = "v${version}";
+    sha256 = "sha256-+q95w1UMCEZeYeXvlLagixUAROakdJFrIBy1rPMyYZc=";
+  };
+
+  format = "pyproject";
+
+  playwright-driver = stdenv.mkDerivation {
+      name = "playwright-driver";
+      src =
+        let
+        inherit (stdenv.hostPlatform) system;
+        selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
+        suffix = selectSystem {
+            x86_64-linux = "linux";
+            aarch64-linux = "linux-arm64";
+            x86_64-darwin = "mac";
+            aarch64-darwin = "mac-arm64";
+        };
+        sha256 = selectSystem {
```

Comment:
I need some advice on these hashes, as MS republishes to the same URL but with updated content meaning these will be invalidated rather frequently.